examples/run_models/configs/geneformer_config.yaml

```diff
@@ -1,3 +1,4 @@
+model_name: "gf-12L-95M-i4096"
 batch_size: 24
```

Comment:
the small model should be default in the examples

Reply:
done